inst/WORDLIST

```diff
@@ -0,0 +1,122 @@
+ALLUPPERCASE
+Addins
+Backport
+CMD
+Checkstyle
+Config
+Ctrl
+Cyclomatic
+Deprecations
+EQ
+ESS
+Github
+Linter
+Linters
+MegaLinter
+ORCID
+RMarkdown
+RStudio
+Rhtml
+Rmarkdown
+Rmd
+Rnw
+Rproj
+Rrst
+Rstudio
+Rtex
+Rtxt
+SARIF
+SublimeLinter
+Tidyverse
+UpperCamelCase
+Wickham
+XPath
+XPaths
+YAML
+Yihui
+addin
```

Comment:
is this an accepted spelling? I would have expected add-in

Reply:
Yes: https://rstudio.github.io/rstudioaddins/